File: app/src/main/java/com/utaste/MainActivity.java
package com.utaste;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // Charge ton layout XML
        setContentView(R.layout.activity_main);
    }
}

File: app/src/main/java/com/utaste/MainActivity.kt
package com.utaste

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        // Charge ton layout XML
        setContentView(R.layout.activity_main)
    }
}